src/WebJobs.Script/Host/AggregateFunctionMetadataProvider.cs

```diff
@@ -62,17 +62,21 @@ public async Task<ImmutableArray<FunctionMetadata>> GetFunctionMetadataAsync(IEn
                     // get function metadata from worker, then validate it
                     rawFunctions = await _dispatcher.GetWorkerMetadata();
 
-                    if (!IsNullOrEmpty(rawFunctions))
+                    if (IsDefaultIndexingRequired(rawFunctions))
```

Comment:
same here, can we rename to `HostIndexingRequired`

Reply:
Done